src/components/keypad.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import '../styles/calculator.css';
+
+function Keypad() {
+  return (
+    <div className="keypad">
+      <button className="key" type="button">AC</button>
+      <button className="key" type="button">+/-</button>
+      <button className="key" type="button">%</button>
+      <button className="key orange" type="button">/</button>
```

Comment:
- [ ] **[OPTIONAL]** This division sign should be &div;, not /, based on the design. You can use HTML entity code if you don't have the sign on your keyboard.